### PR TITLE
[cleanup]Enable no-floating-promises in celotool

### PR DESCRIPTION
**Pull request by ashishb**
### Description

Enable no-floating-promises in celotool. Fix a few bugs which were caught due to this.

### Tested

`packages/celotool $ yarn lint`

### CODE REVIEW

File: packages/celotool/src/cmds/geth/init.ts [hunk 1]
```diff
@@ -73,6 +73,7 @@ export const handler = async (argv: InitArgv) => {
 
   if (argv.fetchStaticNodesFromNetwork) {
     await switchToClusterFromEnv(false)
+    /* tslint:disable-next-line */
```
Comment (nambrot) [inline packages/celotool/src/cmds/geth/init.ts]: Shouldn't we `await` here?

**Review by nambrot — DISMISSED** [review]
Awesome catch, just one point I think

File: packages/celotool/src/cmds/geth/init.ts (hunk 1 above)
Comment (ashishb) [reply packages/celotool/src/cmds/geth/init.ts]: Fixed.

**Review by aaronmgdr — APPROVED** [review]
Trusting @nambrot.

**Pull request merged**